.github/workflows/build.yml

```diff
@@ -1,5 +1,10 @@
-name: Build and Deploy
-on: [push, pull_request]
+name: Test and Deploy
+on:
+  pull_request:
+    types: [assigned, opened, reopened, synchronize]
```

Comment:
Are these types all triggers? Because `assigned` seems like a trigger we may not need, if I guess correctly.

Reply:
TBH I just copied those from the last project